website/source/index.html.erb

```diff
@@ -11,7 +11,7 @@ description: |-
       <div class="col-md-offset-2 col-md-8">
         <%= inline_svg "logo-hashicorp.svg", height: 120, class: "logo" %>
 
-        <h1>Deploy and Manage Any Containerized, Legacy, or Batch Application</h1>
+        <h1 class="g-type-display-1">Deploy and Manage Any Containerized, Legacy, or Batch Application</h1>
```

Comment:
```suggestion
        <h2 class="g-type-display-3">Deploy and Manage Any Containerized, Legacy, or Batch Application</h1>
```

- for semantics, i think we should reduce this to an h2 as its a subheading
- for styling purposes, i think we should use `g-type-display-3`. the current version reads as way too big imo. curious to see what kristopher thinks...

Reply:
I can run this by Kristopher and see what he thinks.

My concern about changing this to an `h2` is around accessibility (https://dequeuniversity.com/rules/axe/3.3/page-has-heading-one?application=AxeChrome).  Thoughts?

Reply:
ah i see. so semantically, we'd want the h1 to be "Nomad" here and this subheader to be an h2 - but changing that across sites should just be out of scope for this PR. I'm fine with keeping it h1 for now, knowing we'll be doing a more thorough a11y review in the near future. 

i do want to see us change the `g-type-display-3` though ;)